net-misc/anydesk/anydesk-5.5.0-r1.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit desktop eutils systemd xdg-utils
+
+DESCRIPTION="Feature rich multi-platform remote desktop application"
+HOMEPAGE="https://anydesk.com"
+SRC_URI="amd64? (
+	https://download.anydesk.com/linux/${P}-amd64.tar.gz
+	https://download.anydesk.com/linux/generic-linux/${P}-amd64.tar.gz
+)
+x86? (
+	https://download.anydesk.com/linux/${P}-i386.tar.gz
+	https://download.anydesk.com/linux/generic-linux/${P}-i386.tar.gz
+)"
```

Comment:
```
SRC_URI="amd64? (
		https://download.anydesk.com/linux/${P}-amd64.tar.gz
		https://download.anydesk.com/linux/generic-linux/${P}-amd64.tar.gz
	)
	x86? (
		https://download.anydesk.com/linux/${P}-i386.tar.gz
		https://download.anydesk.com/linux/generic-linux/${P}-i386.tar.gz
	)"
```
I know it looks weird but we always intend one tab inside variables. Check RDEPEND below for example.

Reply:
It doesn't look weird, but was introduced by the last bump and I missed it...will fix.